spec/models/knight_spec.rb

```diff
@@ -1,5 +1,52 @@
 require 'rails_helper'
 
 RSpec.describe Knight, type: :model do
+  describe 'valid_move?' do
+    before(:each) do
+      @user = create(:user)
+      sign_in @user
 
+      @game = create(:game)
```

Comment:
This should not be necessary. When you create a knight all these things are done except for signing in a user. Since there are no validation checks for current users that should also not be needed.

Reply:
I'll try it without this before block.